recipe/meta.yaml

```diff
@@ -17,19 +17,30 @@ source:
   sha256: 732cdb321f7496c712d70b137b54073192e325ce382c46fe3dd13c1ec82034f3  # [win]
 
 build:
-  number: 0
+  number: 1
   binary_relocation:
     - bin/firefox
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - sysroot_linux-64 2.17  # [linux64]
 
 test:
   requires:
     - selenium
     - geckodriver
+    - pytest-azurepipelines
+    - python >=3.6
+  commands:
+    - copy {{ RECIPE_DIR }}\test_selenium.py .  # [win]
+    - copy {{ RECIPE_DIR }}\setup.cfg .  # [win]
+    - cp {{ RECIPE_DIR }}/test_selenium.py {{ RECIPE_DIR }}/setup.cfg .  # [not win]
+    # TODO: may be needed
+    # - set -m && % xattr -r -d com.apple.quarantine $PREFIX/bin/geckodriver  # [osx]
```

Comment:
Shall we delete this from this PR to keep things clean?

Reply:
nah, if we even think it's going to be needed in the future, i think we should keep it. Indeed, i should probably add a link to the canonical page.

Reply:
aaf61f5